src/tool_cache.rs

```diff
@@ -80,6 +80,10 @@ impl ToolCache {
 
                 let version = Version::parse(&release.tag_name).ok().or_else(|| {
                     if !release.tag_name.starts_with('v') {
+                        log::info!(
+                            "Release tag name did not start with 'v'! {}",
+                            release.tag_name
+                        );
```

Comment:
Not sure why I didn't think about this earlier, but this seems like it could be noisy and/or confusing. We'll print this for every release we see, even if it's got nothing to do with the one that the user asked for, right?

Reply:
Yes you're right, I'll make this a lower log level because it's still valuable I think!